src/types/configType.ts

```diff
@@ -6,6 +6,8 @@ type ColorsType = {
 };
 
 export interface ConfigType {
+  aiReactionChannels: string[],
+  openaiApiKey: string,
```

Comment:
Make these optional, add failguard to these. OpenAI is an external service and which costs real credits so please make it optional and add a failsafe before registering it.